packages/turf-polygon-to-linestring/test.js

```diff
@@ -0,0 +1,68 @@
+const test = require('tape');
+const fs = require('fs');
+const path = require('path');
+const load = require('load-json-file');
+const write = require('write-json-file');
+const polyToLineString = require('./');
+
+const directories = {
+	in: path.join(__dirname, 'test', 'in') + path.sep,
+	out: path.join(__dirname, 'test', 'out') + path.sep
+};
+
+const fixtures = fs.readdirSync(directories.in).map(filename => {
+	return {
+		filename,
+		name: path.parse(filename).name,
+		geojson: load.sync(directories.in + filename)
+	};
+});
+process.env.REGEN = true;
+
+test('turf-polygon-to-linestring', t => {
+	for (const {name, geojson} of fixtures) {
+		var results = polyToLineString(geojson)
+		console.log('Input File: ' + name + '.geojson')
+		if (process.env.REGEN){
+			write.sync(directories.out + name + '.json', results);
```

Comment:
The purpose with `write` is to test against it afterwards.

1. Create results with module
2. Save results (with REGEN=true)
3. Test results if the same as saved results (in these tests it's not being tested)